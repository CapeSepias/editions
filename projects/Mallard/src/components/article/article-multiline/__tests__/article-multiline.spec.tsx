import React from 'react'
import TestRenderer, { ReactTestRendererJSON } from 'react-test-renderer'
import { ArticleMultiline } from '..'

describe('ArticleMultiline', () => {
    it('should display a consistent Multiline for Article Headers', () => {
        const component: ReactTestRendererJSON | null = TestRenderer.create(
            <ArticleMultiline />,
        ).toJSON()
        expect(component).toMatchSnapshot()
    })
})
