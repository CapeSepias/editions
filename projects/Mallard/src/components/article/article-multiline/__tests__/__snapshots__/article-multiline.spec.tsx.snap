// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleMultiline should display a consistent Multiline for Article Headers 1`] = `
<RNSVGSvgView
  bbHeight={11}
  bbWidth="100%"
  fill="none"
  height={11}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      undefined,
      null,
      Object {
        "flex": 0,
        "height": 11,
        "width": "100%",
      },
    ]
  }
  width="100%"
>
  <RNSVGGroup
    fill={null}
    fillOpacity={1}
    fillRule={1}
    font={Object {}}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={
      Array [
        "fill",
      ]
    }
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  >
    <RNSVGRect
      fill={
        Array [
          0,
          4288256409,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      height={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      rotation={0}
      rx={0}
      ry={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      width="100%"
      x={0}
      y={0}
    />
    <RNSVGRect
      fill={
        Array [
          0,
          4288256409,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      height={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      rotation={0}
      rx={0}
      ry={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      width="100%"
      x={0}
      y={3}
    />
    <RNSVGRect
      fill={
        Array [
          0,
          4288256409,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      height={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      rotation={0}
      rx={0}
      ry={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      width="100%"
      x={0}
      y={6}
    />
    <RNSVGRect
      fill={
        Array [
          0,
          4288256409,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      height={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      rotation={0}
      rx={0}
      ry={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      width="100%"
      x={0}
      y={9}
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
